source/widgets/UnixWindow.ooc

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (C) 2014 - Simon Mika <simon@mika.se>
+ *
+ * This sofware is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this software. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import DisplayWindow
+use ooc-draw-gpu
+use ooc-math
+use ooc-draw
+use ooc-opengl
+use ooc-x11
```

Comment:
@jonathanudd has successfully compiled and ran Kean on windows, including his win32 window class, so I guess he solved this somehow.